test/e2e/framework/framework.go

```diff
@@ -261,10 +261,10 @@ func (f *Framework) BeforeEach() {
 		var err error
 		var nodeMode NodesSet
 		switch TestContext.GatherKubeSystemResourceUsageData {
-		case "master":
-			nodeMode = MasterNodes
-		case "masteranddns":
-			nodeMode = MasterAndDNSNodes
+		case "controlplane":
+			nodeMode = ControlPlaneNodes
+		case "controlplaneanddns":
```

Comment:
Same here. Better accept "masteranddns" as alias.

Reply:
@pohly  okay thanks for the feedback